tests/test_server_basic.py

```diff
@@ -132,14 +132,14 @@ def test_empty_request(dispatcher_live_fixture):
                    params={},
                 )
 
-    print("content:", c.text)
+    print("test_empty_request content output:", c.text)
 
     jdata=c.json()
 
     assert c.status_code == 400
 
     # parameterize this
-    assert sorted(jdata['installed_instruments']) == sorted(['empty', 'empty-async', 'empty-semi-async']) or \
+    assert sorted(jdata['installed_instruments']) == sorted(['empty', 'empty-async', 'empty-semi-async', 'empty-development']) or \
```

Comment:
This is not generally presented to the user, but should it really list all the instruments?
It could be not so easy to change, so not critical.

Reply:
I am not sure exactly what you mean so it's best that you try to implement the suggestion so we can see the details

Reply:
I double-checked : this list is built using the `importer.instrument_factory_list`, so indeed it would not be so simple to change the logic for building such a list